libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -1100,6 +1100,19 @@
     <output name="out" type="vector4" default="0.0, 0.0, 0.0, 0.0" />
   </nodedef>
 
+  <!--
+    Node: <bump>
+    Offset the surface normal by a scalar value.
+  -->
+  <nodedef name="ND_bump_vector3" node="bump" nodegroup="geometric">
+    <input name="height" type="float" uiname="Height" uisoftmin="0.0" uisoftmax="1.0" value="0" doc="Amount to offset the surface normal." />
+    <input name="scale" type="float" uiname="Scale" uisoftmin="0.0" uisoftmax="1.0" value="1" doc="Scalar to adjust the height amount." />
+    <input name="space" type="string" value="tangent" enum="tangent, object" uniform="true" doc="The space to transform the normal from ('tangent' or 'object'); defaults to 'tangent'" />
```

Comment:
I believe we can remove this `space` input, and assume the normal map is in tangent space, since the `heighttonormal` node is designed to generate a map in this space.  In the latest draft of MaterialX 1.39, the `space` input has been removed from the `normalmap` node, so removing the input should make the `bump` node more forward-looking.